src/components/document/resize-panel-divider.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import DragThumbnailIcon from "../../assets/drag-thumb-icon.svg";
+import LeftRightDragIcon from "../../assets/left-right-drag.svg";
+import "./resize-panel-divider.scss";
+
+interface IProps {
+  isResourceExpanded: boolean
+  onExpandWorkspace: (expand: boolean) => void;
+}
+export const ResizePanelDivider: React.FC <IProps>= ({isResourceExpanded, onExpandWorkspace}) => {
+  const dividerPosition = isResourceExpanded ? {right: "50%"} :  {left: 44};
+  return (
+    <div className="resize-panel-divider" onClick={()=>onExpandWorkspace(false)} style={dividerPosition}>
```

Comment:
`<div className="resize-panel-divider" onClick={() => onExpandWorkspace(false)} style={dividerPosition}>`

Reply:
And as above, if existing CLUE code prefers the version without spacing, leave it.